config/reasoner/dl2op/private-info/rrn.n3

```diff
@@ -0,0 +1,62 @@
+PREFIX : <#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX adres: <https://data.vlaanderen.be/ns/adres#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX crypto: <http://www.w3.org/2000/10/swap/crypto#>
+PREFIX e: <http://eulersharp.sourceforge.net/2003/03swap/log-rules#>
+PREFIX ex: <http://example.org/test#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX func: <http://www.w3.org/2007/rif-builtin-function#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX log: <http://www.w3.org/2000/10/swap/log#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX persoon: <https://data.vlaanderen.be/ns/persoon#>
+PREFIX persoon_loket: <http://data.vlaanderen.be/ns/persoon#>
+PREFIX skos:<http://www.w3.org/2004/02/skos/core#>
+PREFIX string: <http://www.w3.org/2000/10/swap/string#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+
+# Rijksregisternummer
+# Extend identifiers to match OP's model
+# <http://example.org/tst#subject> skos:notation "00010100105".
+{
+    ?identifier skos:notation ?notation.
+    ?notation string:notEqualIgnoringCase "Rijksregisternummer".
+    ?identifier log:uri ?identifierString.
+    (?identifierString "STRUCTURED_IDENTIFIER")  string:concatenation ?hashInput.
+    ?hashInput crypto:md5 ?uuid.
+    ("http://data.lblod.info/id/gestructureerdeIdentificatoren/" ?uuid) string:concatenation ?structuredIdString.
+    ?structuredIdentifier log:uri ?structuredIdString.
+
+    {
+      ?identifier skos:notation ?notation.
+    } e:becomes {
+      ?identifier
+          generiek:gestructureerdeIdentificator ?structuredIdentifier ;
+          skos:notation "Rijksregisternummer" .
+    }.
+
+} => {
+  ?identifier
```

Comment:
Or here, for the type ? Not sure why it's in the main-info config as it should only end up in the privacy graph, but i'll let you put it in the correct place haha

Reply:
I choose to have the exact same rules in all the configs. So the queries - i.e. filters. Can be done using the OP domain language.
- this eases the pain of having to thing how the stuff we don't want in OP is modelled in Loket. It will also help us once Loket fixes a few things. We don't need to change anything, we could clean up the rules that are no longer triggered at our own pace. Everything will keep working as expected
- It's also simple - for me at least - to only have one kind of mapping logic. The performance impact is negligible. Moving and parsing data takes much more time than the actual reasoning.